motr/m0crate/btree_without_tx_seg.c

```diff
@@ -446,7 +468,6 @@ static void btree_node_free(struct m0_be_bnode       *node,
 {
 	/* invalidate header so that recovery tool can skip freed records */
 	node->bt_header.hd_magic = 0;
-	M0_BE_TX_CAPTURE_PTR(btree->bb_seg, tx, &node->bt_header.hd_magic);
```

Comment:
Instead #define M0_BE_TX_CAPTURE_PTR() to a no-op.

Reply:
Added #define as noop.